src/main/java/filter/expression/QualifierType.java

```diff
@@ -93,4 +118,15 @@ private static boolean isSpecialQualifier(String input) {
     public String toString() {
         return name().toLowerCase();
     }
+    
+    /**
+     * Retrieves acceptable inputs of a qualifier 
+     * @return
+     */
+    public String getValidInputs() {
+        String defaultExpectedInput = "string";
+        if (VALID_INPUTS.containsKey(this)) return VALID_INPUTS.get(this);
+        return defaultExpectedInput;
```

Comment:
Instead of having a Map defined and populated elsewhere, can't we use a switch statement inside this method. Seems this is the only place where that Map is used.